src/libtriton/ast/z3/z3ToTritonAst.cpp

```diff
@@ -149,7 +149,7 @@ namespace triton {
           break;
         }
 
-        //case Z3_OP_BSDIV_I:
+        case Z3_OP_BSDIV_I:
```

Comment:
maybe but it must working on the last z3 release. can you provide a poc because it works on my computer.